proto/Makefile

```diff
@@ -7,3 +7,7 @@ protolint:
 .PHONY: protoc
 protoc: protolint
 	buf generate -v
+
+PHONY: generate-docs
+generate-docs:
+	protoc --doc_out=../docs --doc_opt=json,docs.json --experimental_allow_proto3_optional $(wildcard ./api/v1/*.proto)
```

Comment:
should be done with `buf`

Reply:
Is there a way to generate a json description with buf? I want to integrate the docs into the website from a json-file and I haven't found a way to generate it with buf, that's why I used protoc.

Reply:
evtl mit einem davon: https://buf.build/search?query=protoc-gen-doc